lib/auto_api/states/crash_state.ex

```diff
@@ -55,7 +55,7 @@ defmodule AutoApi.CrashState do
           tipped_state: State.property(tipped_state()),
           automatic_ecall: State.property(CommonData.enabled_state()),
           severity: State.property(integer()),
-          impact_zone: State.property(impact_zone())
+          impact_zone: State.multiple_property(impact_zone())
```

Comment:
hmm why it's not `impact_zones` ? 😬 like `incidents`

Reply:
Setting it multiple is not a breaking change for end users but renaming it it is.

We can deprecate it and rename it on the next level (L14)